bedrock/settings/base.py

```diff
@@ -28,6 +28,7 @@ def path(*args):
 
 # Is this a dev instance?
 DEV = config('DEV', cast=bool, default=False)
+PROD = config('PROD', cast=bool, default=False)
```

Comment:
Not sure if this is the way to go, but like we discussed the other day, we could set this in all the prod envs once, then use it for other things like this. I'm open to other suggestions though for sure.